device-registry/src/main/scala/org/genivi/sota/device_registry/Routes.scala

```diff
@@ -122,48 +120,45 @@ class Routes(namespaceExtractor: Directive1[Namespace],
   def api: Route =
```

Comment:
After having a look, I think it probably makes sense to split system_info and group_info into separate prefixes, after "devices". It would also then make sense to change the url structure on web. So I think making a separate ticket for it makes sense, as it will be a fairly big change.